wiremock/mappings/ndelius.json

```diff
@@ -0,0 +1,65 @@
+{
+  "mappings": [
+    {
+      "request": {
+        "url": "/delius-api/case-details/NOTFOUND",
+        "method": "GET"
+      },
+      "response": {
+        "status": 404
+      }
+    },
+    {
+      "request": {
+        "url": "/delius-api/case-details/SERVERERROR",
+        "method": "GET"
+      },
+      "response": {
+        "status": 500
+      }
+    },
+    {
+      "request": {
+        "urlPattern": "/delius-api/case-details/(?!NOTFOUND$|SERVERERROR$).*",
+        "method": "GET"
+      },
+      "response": {
+        "status": 200,
+        "headers": { "Content-Type": "application/json" },
+        "jsonBody": {
+          "crn": "X123456",
+          "pnc": "01/14106572A",
+          "nomisId": "",
+          "givenName": "Buster",
+          "familyName": "Sanford",
+          "dateOfBirth": "2002-01-15",
+          "gender": 1,
+          "location": "COMMUNITY",
+          "sexuallyMotivatedOffenceHistory": "YES",
+          "sentences": [
+            {
+              "description": "yay",
```

Comment:
🎉

Reply:
i'm going to leave this for the moment :D